### PR TITLE
converted some to es6

### PR DESCRIPTION
**Pull request by Rafi993**
I did convert some code to es6 but I think it can be done in  more elegent way.

### CODE REVIEW

**Comment by jlongster** [comment]
Are all recent Atom version guaranteed to have support for this?

**Comment by Rafi993** [comment]
yes they do. Even the official tutorial https://github.com/blog/2231-building-your-first-atom-plugin is written in es6

File: lib/prettier.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /* global atom */
 
-var path = require("path");
-var findRoot = require("find-root");
-var prettier = require("prettier");
+const path = require("path"),
+ findRoot = require("find-root"),
+ prettier = require("prettier");
```
Comment (jlongster) [inline lib/prettier.js]: Can you do a `const` per line? That's why I prefer.

File: lib/prettier.js [hunk 2]
```diff
@@ -15,12 +15,12 @@ module.exports = {
     "trailingComma",
     "bracketSpacing"
   ],
-  fileSupported: function(file) {
+  fileSupported: (file)=> {
     // Ensure file is a supported file type.
     var ext = path.extname(file);
```
Comment (robwise) [inline lib/prettier.js]: should this be a `const`?

File: lib/prettier.js [hunk 3]
```diff
@@ -33,35 +33,35 @@ module.exports = {
       this.handleEvents.bind(this)
```
Comment (robwise) [inline lib/prettier.js]: I think we can remove this line because `handleEvents` is now bound via fat arrow on line 77

**Comment by jlongster** [comment]
We might should go ahead and merge this and do a follow-up. It's going to conflict easily with other ones.

**Comment by robwise** [comment]
Closing this as it is superseded by #22 (which your work inspired 🥇)

**Pull request closed without merging**